source/lmp/fix_dplr.cpp

```diff
@@ -62,6 +62,7 @@ FixDPLR::FixDPLR(LAMMPS *lmp, int narg, char **arg)
   size_vector = 3;
   qe2f = force->qe2f;
   xstyle = ystyle = zstyle = NONE;
+  pair_deepmd_index = 0;
```

Comment:
should the default be 1?

Reply:
0 and 1 have different meanings:
- 0: If there is one pair, that pair will be found; if there are two pairs, no pair will be found;
- 1: The first pair is always found.

I feel that "if there are two pairs, no pair will be found" is safer. Users have to explicitly assign an index.